Json.More/JsonSerializerOptionsExtensions.cs

```diff
@@ -19,7 +19,7 @@ public static class JsonSerializerOptionsExtensions
 	/// <param name="options">The <see cref="JsonSerializerOptions"/> being used.</param>
 	/// <param name="typeInfo">An explicit typeInfo to use for looking up the Converter. If not provided, options.GetTypeInfo will be used.</param>
 	/// <returns>An implementation of <see cref="JsonConverter{T}"/> as determined by the provided options</returns>
-	public static JsonConverter<T> GetConverter<T>(this JsonSerializerOptions options, JsonTypeInfo? typeInfo = null)
+	public static JsonConverter<T> GetConverter<T>(this JsonSerializerOptions options, JsonTypeInfo? typeInfo)
 	{
 #if NET8_0_OR_GREATER
```

Comment:
It seems that targets older than 8.0 ignore the `typeInfo` argument?

Reply:
Yeah, .Net Standard's version doesn't have a `.Converter` property on `JsonTypeInfo`, so we need to just get it off of the options.

Reply:
Gotcha. Would it make sense to add a debug assertion?
```C#
Debug.Assert(typeInfo is null || typeInfo.Options == options);
```
Or perhaps even make it a runtime check given that it's public API?